catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/CategoryListFragment.java

```diff
@@ -312,6 +317,51 @@ public void onItemClick(CategoryListItem item) {
 		}
 	}
 
+	@Override
+	public boolean onOptionsItemSelected(MenuItem item) {
+		switch (item.getItemId()) {
+			case R.id.wiki_help:
+				onOptionsMenuClick(getTag());
+				break;
+		}
+		return true;
+	}
+
+	public void onOptionsMenuClick(String tag) {
+		switch (tag) {
+			case FUNCTION_TAG:
+				startActivity(new Intent(Intent.ACTION_VIEW,
+						Uri.parse(Constants.CATROBAT_FUNCTIONS_WIKI_URL)));
+				break;
+			case LOGIC_TAG:
+				startActivity(new Intent(Intent.ACTION_VIEW,
+						Uri.parse(Constants.CATROBAT_LOGIC_WIKI_URL)));
+				break;
+			case OBJECT_TAG:
+				startActivity(new Intent(Intent.ACTION_VIEW,
+						Uri.parse(Constants.CATROBAT_OBJECT_WIKI_URL)));
+				break;
+			case SENSOR_TAG:
+				startActivity(new Intent(Intent.ACTION_VIEW,
+						Uri.parse(Constants.CATROBAT_SENSORS_WIKI_URL)));
+				break;
+		}
+	}
+
+	public String getHelpUrl(String tag) {
+		switch (tag) {
+			case FUNCTION_TAG:
+				return Constants.CATROBAT_FUNCTIONS_WIKI_URL;
+			case LOGIC_TAG:
+				return Constants.CATROBAT_LOGIC_WIKI_URL;
+			case OBJECT_TAG:
+				return Constants.CATROBAT_OBJECT_WIKI_URL;
+			case SENSOR_TAG:
+				return Constants.CATROBAT_SENSORS_WIKI_URL;
```

Comment:
Please also include the corresponding language to your helpurl from the sharedpreferences (see settingsfragment for getting language tag)